commons/active/questionnaire/questionnaire_completion_log.avsc

```diff
@@ -0,0 +1,11 @@
+{
+  "namespace": "org.radarcns.active.questionnaire",
+  "type": "record",
+  "name": "QuestionnaireCompletionLog",
+  "doc": "Schema for reporting the completion status of a questionnaire. This will help in calculating the compliance.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Timestamp in UTC (s) when the questionnaire completion log is submitted." },
+    { "name": "name", "type": "string", "doc": "Questionnaire name." },
+    { "name": "completionPercentage", "type": "double", "doc": "Percentage of the questionnaire completed. 0 for not at all complete and 100 for full completion."}
```

Comment:
I wonder if we should provide the option to include a NULL value say completion is not a value for a question type or we in the future include aRMT questions where there is no logical completion value (e.g. a hidden field)?